packages/vite/src/node/__tests__/plugins/importGlob/fixture-a/index.ts

```diff
@@ -5,21 +5,43 @@ export interface ModuleType {
 }
 
 export const basic = import.meta.glob<ModuleType>('./modules/*.ts')
+export const basicWithObjectKeys = Object.keys(
+  import.meta.glob<ModuleType>('./modules/*.ts'),
+)
```

Comment:
```suggestion
// prettier-ignore
export const basicWithObjectKeys = Object.keys(import.meta.glob<ModuleType>('./modules/*.ts'))
```
It would be nice if we have a test case without a line break.